tests/backwards_compatibility_tests/scripts/generate_versions.py

```diff
@@ -24,13 +24,16 @@ def generate_versions(to_version: str, num_versions: int = 3) -> list:
 
     # If this is a patch release, add the previous patch version of the same minor version
     if target_version.patch > 0:
-        prev_patch_version = f"{target_version.major}.{target_version.minor}.{target_version.patch - 1}"
-        versions.append(prev_patch_version)
+        versions.extend(
```

Comment:
I think we just need to test the previous 1 patch version (tip of the minor releases). Just like mainline only test the latest patch version of each minor version.

Reply:
@farshidz , why would we test all previous patch versions?